app/flatpages/privacy_policy.md

```diff
@@ -32,29 +33,40 @@ Please avoid giving other personal information as we do not want it.
     * Provide essential system messages, such as password resets.
     * Send notifications - the user may configure this to their needs, including opting out.
 * Passwords are used to authenticate the user.
+* The audit log is used to record actions that may be harmful
 * Other information is displayed as part of ContentDB's service.
 
 ## Who has access
 
 * Only the admin has access to the HTTP requests.
   The logs may be shared with others to aid in debugging, but care will be taken to remove any personal information.
+* Encrypted backups are shared with Minetest staff members.
```

Comment:
Who is `staff`? Where is it documented?

Reply:
Moderators and core developers. I currently don't do this with anyone, this is to allow it to be done in the future

Reply:
```suggestion
* Encrypted backups may be shared with select Minetest staff members (moderators + core devs).
```